scripts/ornl.frontier.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+module load cray-python
+pip install --target=`pwd`/../install radical.repex radical.analytics
```

Comment:
probably we don't need `repex` at this moment, but rather RP, RE and RA,
AND we need to ensure versions:
- should we name Lua-file with a range of versions? -> `1.8X.lua`
- and then ensure that the corresponding range of versions is set per each package
   ```
    'radical.utils>=1.80,<1.90' \
    'radical.gtod>=1.80,<1.90' \
    'radical.pilot>=1.80,<1.90' \
    'radical.entk>=1.80,<1.90' \
    'radical.analytics>=1.80,<1.90'  
   ```

Reply:
one more thing - would it help to use the following env variable to avoid installations in a user space?
```
export PYTHONNOUSERSITE=True
```

Reply:
the --target overrules the usersite installation anyway, so that is not needed.  And I also would avoid setting globally in the module as that is not needed and unexpected for the user.

Reply:
> I also would avoid setting globally in the module

agree, also didn't mean that, just to have it within the script or even per command (e.g., `PYTHONNOUSERSITE=True pip install ...`)

Reply:
I added the version specs now, but I don't think we should use `1.8X.lua` - more likely than not that would trip some version string parser eventually.  I set it to `1.80.lua` for now - but we may want to discuss version numbers again.  

A bit off topic: it might be good to change from the scheme `1.80, 1.81, 1.82, 1.90, 1.91, ...` to `1.80.0, 1.80.1, 1.80.2, 1.81.0, 1.81.1, ...`.  That changes the version spec to `radical.pilot~=1.80` for example, and avoids naming shenanigans like `1.8X`.

Reply:
Couple comments

1) should we still use `PYTHONNOUSERSITE` env variable during the installation? e.g., `PYTHONNOUSERSITE=True pip install ...`

2) _this comment can be converted into a ticket, and be resolved outside of this PR_

For frontier it requires to have a virtual env where RP is installed
```$ cat pmgr_launching.0000.log
1730405272.345 : pmgr_launching.0000  : 2700173 : 140569850410752 : ERROR    : virtenv_mode is local, no local env found
1730405272.350 : pmgr_launching.0000  : 2700173 : 140569850410752 : ERROR    : bulk launch failed
Traceback (most recent call last):
  File "/lustre/orion/world-shared/chm155/radical/radical.modules/install/radical/pilot/pmgr/launching/base.py", line 297, in work
    self._start_pilot_bulk(resource, schema, pilots)
  File "/lustre/orion/world-shared/chm155/radical/radical.modules/install/radical/pilot/pmgr/launching/base.py", line 399, in _start_pilot_bulk
    self._prepare_pilot(resource, rcfg, pilot, expand, tar_name)
  File "/lustre/orion/world-shared/chm155/radical/radical.modules/install/radical/pilot/pmgr/launching/base.py", line 734, in _prepare_pilot
    raise ValueError('no local env found')
ValueError: no local env found
```

Reply:
This is now addressed in https://github.com/radical-cybertools/radical.pilot/pull/3252.

Reply:
> should we still use PYTHONNOUSERSITE env variable during the installation? e.g., `PYTHONNOUSERSITE=True pip install ...`

No - I am not a fan of setting random env vars just because they sound nice ;-)